src/EagerLoadedDataList.php

```diff
@@ -213,12 +214,14 @@ public function eagerLoadManyMany(&$data, $manyManys, $withManyManys)
                 'map' => [],
             ];
 
+            $table = DataObject::getSchema()->tableName($component['join']);
```

Comment:
This breaks existing functionality.
I suggest
```php
$table = isset($depClass['through'])
                ? DataObject::getSchema()->tableName($depClass['through'])
                : $component['join'];
```

Reply:
Fixed, thanks! Just tested with two standards many_many and two many_many through.